packages/react-router-website/modules/components/FakeBrowser/index.js

```diff
@@ -61,7 +61,7 @@ class FakeBrowser extends React.Component {
 
     return (
       <MemoryRouter getUserConfirmation={getUserConfirmation}>
-        <Route render={({ canGo, goBack, goForward, push, location }) => (
+        <Route render={({ history, location }) => (
```

Comment:
I had to fix a few things in the examples to get them working with the new `{ match, location, history }` props.